src/shared/components/ncTable/partials/columnTypePartials/forms/MainForm.vue

```diff
@@ -95,6 +109,12 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		customSettings: {
+			type: [Object, Array],
```

Comment:
If I understand correctly, the Array type is when it is empty. Can you correct it so that the type is only Object?

Reply:
It will not work as expected then because server responds with `[]` instead of `{}` if width not defined yet.

I had an attempt to return `new \stdClass()` to always return `{}` but in this case we can't properly serialize response.

Reply:
ok, I managed to fix it